### PR TITLE
Prepare for version 2 1

### PR DESCRIPTION
**Pull request by tasomaniac**

### CODE REVIEW

**Comment by hal9002** [comment]
Can one of the admins review this PR?

File: README.md [hunk 1]
```diff
@@ -217,23 +217,6 @@ install {
 
 More flags can be found in the `install` section of [the official adb document](https://developer.android.com/studio/command-line/adb.html#pm).
 
-**deviceId**
-
-*Note:* Due to a [bug](https://github.com/novoda/gradle-android-command-plugin/issues/138), setting `deviceId` has been disabled for version `2.0.1`.
```
Comment (tasomaniac) [inline README.md]: This was changed for 2.0.1 but now removed completely. I don't think there is a good way to support this feature. `deviceId` can already be overridden overall.

**Comment by tasomaniac** [comment]
@mr-archano can you have a look and do a release? Thanks

**Comment by mr-archano** [comment]
@hal9002 retest this please

**Pull request merged**